tests/integration/test_composable_protocols/test.py

```diff
@@ -0,0 +1,69 @@
+import ssl
```

Comment:
We need to add tests with incorrect configs

Reply:
it doesn't make much sense - server will just not start

Reply:
Well, my PoV - we've added some checks about config correctness for layer protocol, which would be nice to test. You to decide